### ISSUE TITLE
Bot doesn't relay messages 

### ISSUE THREAD

**Issue by Felagund99** [reactiflux/discord-irc#557]
Hello,

I have two problems with relaying the messages. Firstly, when I type in Discord the bot doesn't relay it to IRC and nothing shows up in my command prompt. Second, when I try to relay a message from IRC to Discord, nothing pops up in Discord but I do get an error in my command prompt:

2020-08-11T23:52:00.574Z info: Tried to send a message to a channel the bot isn't in:  {
  level: undefined,
  message: undefined,
  timestamp: '2020-08-11T23:52:00.574Z',
  splat: undefined,
  [Symbol(level)]: 'info',
  [Symbol(splat)]: [ '#CHANNEL ID' ]

Anyone got a suggestion how to fix this:?

**Comment by Throne3d** [comment]
Could you show a snippet of your config? Channel IDs shouldn't have a `#` before them, if they're the numeric ID, but I'm not sure if that's really the problem here.

**Comment by Max-Rodriguez** [comment]
Yeah I have my bot running with the proper configuration (using a Channel ID, no `#` in the string value for it) and it's not relaying discord messages over to IRC, but it is relaying IRC messages to discord.

**Comment by algoatson** [comment]
same issue here

**Comment by yowipo1** [comment]
I'm having this issue.  Messages from IRC appear on Discord just fine.  Messages from Discord not appear on IRC.  Should I suspect the IRC server? Besides EFnet, I also tried on Libera.Chat.  Same thing.  No error on the console.
```
[
  {
    "nickname": "untitled",
    "server": "irc.efnet.org",
    "discordToken": "mytoken",
    "channelMapping": {
      "13[...]34": "#untitled"
    }
  }
]
```
```
$ npm start -- --config untitled-libera.conf 

> discord-irc@2.9.0 start
> node dist/index.js --config untitled-libera.conf

2025-01-09T12:53:19.195Z info: Connected to Discord
2025-01-09T12:53:19.902Z info: Connected to IRC
```
(I removed the token and channel id in the mapping.)
